cmd/snap/cmd_login.go

```diff
@@ -111,6 +111,8 @@ func (x *cmdLogin) Execute(args []string) error {
 		return ErrExtraArgs
 	}
 
+	fmt.Fprint(Stdout, i18n.G("Authenticated operation on this device will be under terms of service found at: https://www.ubuntu.com/legal/<some-snap-store-url>")+"\n\n")
```

Comment:
I haven't heard an update on wording but the URL is https://www.ubuntu.com/legal/dataprivacy/snap-store

I'll follow up